client/www/index.html

```diff
@@ -86,6 +87,7 @@
 
 <!-- moment -->
 <script src="./scripts/vendor/moment.js"></script>
+<script src="./scripts/vendor/angular-moment.js"></script>
 
```

Comment:
the angular-moment is a directive that allows you to use moment inside angular views